.github/workflows/pull_requests.yaml

```diff
@@ -162,3 +162,13 @@ jobs:
             --skip-missing-values \
             --excluded-charts console \
             --target-branch ${{ github.event.repository.default_branch }}
+  summary:
+    if: always()
+    needs:
+      - test
```

Comment:
```suggestion
    needs:
      - test
      - lint
```

Reply:
No, those can be checked directly. Only the matrix tests need this.